custom_containers/README.md

```diff
@@ -0,0 +1,5 @@
+# Training with Custom Containers
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+
+Cloud Machine Learning Engine recently added support for training models using user provided docker containers. 
```

Comment:
nit: Is there a public link where user can go to find other sample containers?

Reply:
This README is a dummy one. This sample will be used in the public doc and we can point to the doc later once we have the doc change published. @thedriftofwords can confirm.